app/assets/images/payment_icons/cash.svg

```diff
@@ -1 +1,2 @@
-<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-cash" width="38" height="24" role="img"><title id="pi-cash">Cash</title><path fill="#98C74C" d="M38 21a3 3 0 0 1-3 3H3a3 3 0 0 1-3-3V3a3 3 0 0 1 3-3h32a3 3 0 0 1 3 3v18z"/><circle opacity=".2" cx="19" cy="12" r="8"/><circle opacity=".2" cx="32" cy="12" r="2"/><circle opacity=".2" cx="6" cy="12" r="2"/><path fill="#fff" d="M22.113 13.897c0-1.573-1.271-2.067-2.518-2.42V8.763c.65.118 1.218.588 1.68 1.105l.71-.797c-.578-.609-1.424-1.128-2.39-1.246V6.83h-1.034v.973c-1.408.154-2.413 1.036-2.413 2.275 0 1.461 1.2 1.907 2.413 2.244v3.024c-.898-.145-1.608-.759-2.053-1.277l-.62.945c.597.673 1.504 1.151 2.674 1.266v.89h1.034v-.881c1.819-.186 2.517-1.304 2.517-2.392zm-4.974-3.948c0-.684.627-1.11 1.422-1.223v2.459c-.797-.133-1.422-.596-1.422-1.236zm2.456 5.426v-2.762c.813.254 1.451.59 1.451 1.359 0 .565-.364 1.246-1.451 1.403z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-{cash}">
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-cash">
```

This needs to match the `id` set on the `<title>` tag to create the right connection.